Assets/Thirdweb/Examples/Scripts/Prefab_ConnectWallet.cs

```diff
@@ -0,0 +1,256 @@
+    {
+        Wallet wallet;
+        if (Enum.TryParse<Wallet>(_walletStr, out wallet))
+            OnConnect(wallet);
+        else
+            LogThirdweb($"Did not find wallet: {_walletStr}");
+    }
+
+    public void OnClickDropdown()
+    {
+        dropdownExpand = !dropdownExpand;
+
+        if (dropdownRoutine != null)
+            StopCoroutine(dropdownRoutine);
+
+        dropdownRoutine = StartCoroutine(DropdownRoutine(String.IsNullOrEmpty(address) ? connectDropdown : connectedDropdown));
+    }
+
+    IEnumerator DropdownRoutine(Transform _object)
+    {
+        _object.gameObject.SetActive(true);
+        Vector3 targetScale = dropdownExpand ? Vector3.one : Vector3.zero;
+        while (Math.Abs(_object.localScale.x - targetScale.x) > 0.01f)
+        {
+            _object.localScale = Vector3.Lerp(_object.localScale, targetScale, 10f * Time.deltaTime);
+            yield return new WaitForEndOfFrame();
+        }
+        _object.localScale = targetScale;
+        _object.gameObject.SetActive(dropdownExpand);
+    }
+
+    // Utility
+
+    int GetChainID(Chain _chain)
+    {
+        switch (chain)
+        {
+            case Chain.Ethereum:
+                return 1;
+            case Chain.Goerli:
+                return 5;
+            case Chain.Polygon:
+                return 137;
+            case Chain.Mumbai:
+                return 80001;
+            case Chain.Arbitrum:
+                return 42161;
+            case Chain.ArbitrumGoerli:
+                return 421613;
+            case Chain.Avalanche:
+                return 43114;
+            case Chain.AvalancheFujiTestnet:
+                return 43113;
+            case Chain.BinanceSmartChainMainnet:
+                return 56;
+            case Chain.BinanceSmartChainTestnet:
+                return 97;
+            case Chain.Fantom:
+                return 250;
+            case Chain.FantomTestnet:
+                return 4002;
+            case Chain.Optimism:
+                return 10;
+            case Chain.OptimismGoerli:
+                return 420;
+            default:
+                throw new UnityException($"Chain ID for chain {_chain} unimplemented!");
+        }
+    }
+
+    WalletProvider GetWalletProvider(Wallet _wallet)
+    {
+        switch (_wallet)
+        {
+            case Wallet.MetaMask:
+                return WalletProvider.MetaMask;
+            case Wallet.CoinbaseWallet:
+                return WalletProvider.CoinbaseWallet;
+            case Wallet.WalletConnect:
+                return WalletProvider.WalletConnect;
+            case Wallet.Injected:
+                return WalletProvider.Injected;
+            case Wallet.MagicAuth:
+                return WalletProvider.MagicAuth;
+            default:
+                throw new UnityException($"Wallet Provider for wallet {_wallet} unimplemented!");
+        }
+    }
+
+    void LogThirdweb(string _message)
+    {
+        Debug.Log($"[Thirdweb] {_message}");
```

Comment:
one thing that ive been thinking of is a global flag to enable / disable logs in the SDK. Wonder if these might get a bit verbose? fine for now tho

Reply:
Having logging as an option is probably a good idea, would remove a bunch of logs the dev would typically add anyway.